api_app/analyzers_manager/migrations/0071_analyzer_config_greynoise_labs.py

```diff
@@ -0,0 +1,135 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": {
+            "minute": "0",
+            "hour": "*",
+            "day_of_week": "*",
+            "day_of_month": "*",
+            "month_of_year": "*",
+        },
+        "module": "greynoise_labs.GreynoiseLabs",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "name": "Greynoise_Labs",
+    "description": "scan an IP against the Greynoise Labs API (requires authentication token obtained from cookies on greynoise website)",
```

Comment:
please add markdown with link to the service here so it will be displayed in the gui